mlblocks/mlblock.py

```diff
@@ -110,6 +110,33 @@ def _extract_params(self, kwargs, hyperparameters):
 
         return init_params, fit_params, produce_params
 
+    @staticmethod
+    def _filter_conditional(conditional, init_params):
+        condition = conditional['condition']
+        if condition not in init_params:
+            return conditional
+
+        condition_value = init_params[condition]
+        values = conditional['values']
+        conditioned = values.get(condition_value) or values.get('*')
+        if conditioned:
```

Comment:
This if statement is not needed, as `conditioned` can have only two possible values:
- Contents from the key `condition_value` or `'*'`
- `None`

If we don't make a return when conditioned is `None`, python will interpret that an implicit `return None`, so we could simply replace lines 121-123 with:

``` python
return values.get(condition_value) or values.get('*')
```